app/models/city.rb

```diff
@@ -0,0 +1,4 @@
+class City < ApplicationRecord
+	has_many :students
+	has_many :choices
```

Comment:
https://pre-commit.com/

Once installed and configured, it will run pre-commit (and any other configured hook) each time you commit. If there are any offences, the commit won't be saved

Here is our config:
```
repos:
  - repo: git://github.com/pre-commit/pre-commit-hooks
    rev: v4.0.1
    hooks:
      - id: trailing-whitespace
        exclude: vendor/|.js.snap|.yml.enc
      - id: check-yaml
        exclude: spec/support/|.kubernetes|.drone.yml
      - id: end-of-file-fixer
        exclude: vendor/|.yml.enc
      - id: check-added-large-files
        args: [--maxkb=1024]
  - repo: git://github.com/mattlqx/pre-commit-ruby
    rev: v1.3.3
    hooks:
      - id: rubocop
        args: [-a, --force-exclusion]

```